### DISCUSSION TITLE
How to add Inline Text Expansion for Abbreviations/Explanations?

### DISCUSSION THREAD

**Discussion by helloqxwang** [alshedivat/al-folio#2901] [Q&A] [answered]
Hi! I am new here and thanks so much for the community to build such a beautiful and useful academic website template. 
I am wondering how to add an "inline expansion" feature in the text, which means when I hover/click on a certain text, the text can be expanded into a more detailed version. This will be very useful for extending abbreviation or explanation of a specific term in my research interests.  For example, extend "IL" into "Imitation Learning" in my [website](https://helloqxwang.github.io/).
I found this kind of thing on someone else website but I don't know how to write this in an al-folio template. Thanks so much in advance!

**Comment by george-gca** [comment, marked as answer]
This is not possible with markdown alone, you'll have to use html/css for it. Take a look at [our code](https://github.com/alshedivat/al-folio/blob/c5101beae4af04207ce99b55acb78f58983d4d2b/_layouts/bib.liquid#L136) and how this is done in the [selected publication](https://github.com/alshedivat/al-folio/blob/c5101beae4af04207ce99b55acb78f58983d4d2b/_bibliography/papers.bib#L59) in the [main page](https://alshedivat.github.io/al-folio/) of our demo. I believe this is the kind of thing you want to achieve.
